paddle/fluid/operators/conv_op.cc

```diff
@@ -131,6 +127,22 @@ void Conv2DOpMaker::Make() {
            "The format of output tensor is X (one-dimensional) of size equal"
            "to the number of output channels. Only used with MKL-DNN.")
       .AsDispensable();
+  AddInput("Scale_in",
+           "(Tensor) Scale_in to be used for int8 input data."
+           "Only used with INT8.")
+      .AsDispensable();
+  AddInput("Scale_in_eltwise",
+           "(Tensor) Scale_in_eltwise to be used for int8 eltwise input data."
+           "Only used with MKL-DNN.")
+      .AsDispensable();
+  AddInput("Scale_weights",
+           "(Tensor) Scale_weights to be used for int8 weights data."
+           "Only used with MKL-DNN.")
+      .AsDispensable();
+  AddInput("Scale_out",
+           "(Tensor) Scale_out to be used for int8 output data."
+           "Only used with MKL-DNN.")
+      .AsDispensable();
```

Comment:
Discussed with @qingqing01, for line 130-145, to avoid more and more input/output only for MKLDNN, could you create a new OP such as conv_fusion_op.cc?

Reply:
It could of one of the design options; this PR follows our design reviewed and approved in end of Sep. We have the plan to create a new OP in later PRs.

Reply:
> We have the plan to create a new OP in later PRs

Could I ask when will you to create a new OP? 

Otherwise, how do you use  `Scale_out` input in python-side? As far as I know, if you want to use `Scale_out` input in python-side, you must change the `conv2d` api interface in `nn.py` and `API.spec`.

https://github.com/PaddlePaddle/Paddle/blob/4f71a6ee2c0225e46c1dcb65b09a3aa2f574494b/paddle/fluid/API.spec#L72

https://github.com/PaddlePaddle/Paddle/blob/4f71a6ee2c0225e46c1dcb65b09a3aa2f574494b/python/paddle/fluid/layers/nn.py#L1688-L1699